sympy/core/sympify.py

```diff
@@ -28,9 +28,13 @@ def __str__(self):
             str(self.base_exc)))
 
 
-# See sympify docstring.
 converter = {}  # type: tDict[Type[Any], Callable[[Any], Basic]]
 
+#holds the conversions defined in SymPy itself, i.e. non-user defined conversions
+_sympy_converter = {} # type: tDict[Type[Any], Callable[[Any], Basic]]
+
+#alias for clearer use in the library
+_external_converter = converter
```

Comment:
It would be good to add some explanation here for why there are two converters and which should be used within the codebase.